### PR TITLE
Add Amazon Cloudwatch Evidently Modules

### PR DESCRIPTION
**Pull request by oharaandrew314**

### CODE REVIEW

File: amazon/evidently/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Secrets Manager
```
Comment (daviddenton) [inline amazon/evidently/README.md:1]: Wrong name! 😏

File: amazon/README.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ http4k-connect provides a standardised mechanism to connect to the following AWS
 - SQS
 - STS
 - SystemsManager
+- Evidently
```
Comment (daviddenton) [inline amazon/README.md:17]: The ordering is triggering my ocdz 🙃

File: amazon/evidently/README.md (hunk 1 above)
Comment (oharaandrew314) [reply amazon/evidently/README.md]: Guess we both know which module I copy-pasted from.

File: amazon/README.md (hunk 2 above)
Comment (oharaandrew314) [reply amazon/README.md]: 😅

**Pull request merged**